### ISSUE TITLE
Maybe move fuzz into separate repo

### ISSUE THREAD

**Issue by peterhellberg** [jmespath/go-jmespath#17]
The fuzz corpus is included when vendoring this package (using godep for example)
this was probably not intended?

**Comment by BrianBland** [comment]
```
Godep does not copy:
files from source repositories that are not tracked in version control.
*_test.go files.
testdata directories.
```

Maybe we can move the fuzz corpus into a `testdata` directory...

**Comment by jamesls** [comment]
Not intentional, thanks for letting me know.  I think @BrianBland PR seems reasonable.

**Comment by jamesls** [comment]
Merged #19, closing issue.
